src/rendering/webgl/programs/common/program.ts

```diff
@@ -21,6 +20,15 @@ const SIZE_FACTOR_PER_ATTRIBUTE_TYPE: Record<number, number> = {
   [WebGL2RenderingContext.FLOAT]: 4,
 };
 
+function getAttributeItemsCount(attr: ProgramAttributeSpecification): number {
+  return attr.normalized ? 1 : attr.size;
```

Comment:
Can there be cases where the attribute is normalized but the size is larger than one? I don't think we used something of the kind in the past but could this break in the future?